csrc/executor.cpp

```diff
@@ -1861,6 +1861,7 @@ std::vector<at::Tensor> FusionExecutor::evaluateFusionOutputs(
     for (const auto& out_val : fusion()->outputs()) {
       auto out_tensor =
           expr_eval.evaluate(out_val->as<TensorView>()).as<at::Tensor>();
+      expr_eval.bind(out_val, out_tensor);
```

Comment:
Doesn't this happen automatically when you call  `expr_eval.evaluate`?

Reply:
No, binding does not happen automatically. `validateValWithConcrete` is only triggered when `bind` is called. (Discussion: https://github.com/NVIDIA/Fuser/pull/2534#discussion_r1667013063).

I'll close out your PR #2534 after this is merged.

Reply:
Oh right. Haha I was forgetting my own PR. After you merge this PR, we can merge #2534 and remove this line as part of it.

Reply:
I dont't understand. Why do we need to remove this line?

Reply:
Oh, nevermind. This _is_ the line from #2534. You're right; sorry about the confusion. Yes we can go ahead and close out #2534.